__tests__/endpoints.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end())
+
+describe("/api/topics", () => {
+    test(`Get 200 and an array of topic objects. 
```

Comment:
Nice first test! 
Have a think about what errors you might get here too (no need to add anything but worth thinking about)